.github/workflows/publish-subsplit.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Sub-Split Publishing
+on:
+    push:
+        branches:
+            - master
+    create:
+        tags:
+            - '9.*'
+    delete:
+        tags:
+            - '9.*'
+
+jobs:
+    publish_subsplits:
+        runs-on: ubuntu-latest
+        name: Publish package sub-splits
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  fetch-depth: '0'
+                  persist-credentials: 'false'
+            - uses: frankdejonge/use-github-token@1.0.1
+              with:
+                  authentication: 'nyamsprod:${{ secrets.PERSONAL_ACCESS_TOKEN }}'
+                  user_name: 'Ignace Nyamagana Butera'
+                  user_email: 'nyamsprod@gmail.com'
+            - name: Cache splitsh-lite
+              id: splitsh-cache
+              uses: actions/cache@v2
+              with:
+                  path: './.splitsh'
+                  key: '${{ runner.os }}-splitsh'
+            - uses: frankdejonge/use-subsplit-publish@1.0.0-beta.5
+              with:
+                  source-branch: '3.x'
```

Comment:
Is this correct?

Reply:
good catch I did not update that part 👍🏾

Reply:
should be `master` instead if I am correct